spec/utilities/protocol_importers/step_description_builder_spec.rb

```diff
@@ -60,5 +60,11 @@
         expect(described_class.generate(description_with_html).scan('script').count).to be == 0
       end
     end
+
+    context 'when have allowed html tags' do
+      it 'does not strip img tags' do
+        expect(described_class.generate(description_only).scan('img').size).to eq(1)
```

Comment:
> Cool, see comments. I'm wondering how the errors will be handled here. Will this be done after @mlorb PR or how will they be passed?

I was not thinking about handling and passing any errors on this level. What kind of errors you are worried about here? Not valid HTML? 
We can't guarantee about getting perfect/ok HTML from API, so I don't see a big benefit to validating it HTML strictly. 
Currently, we are striping everything but some tags (what we have in Constants).
If anything is built strange, user can change it inside of RTE.  

Sure, we can take a look at `Sanitize` gem if there are any advantages to built-in sanitizer.

Reply:
Just thinking out loud. :) I mean errors from  `TablesBuilder`, `AttachmentsBuilder`, where something can go wrong, while acting upon data and not HTML per se. It probably should be rescued by the normalizer anyway.

Reply:
Ok, thanks, good point. 
I assumed all "wrong" things will be handled at the end with AR validations, and I assumed we have everything to build `new` records. 
The only problem that came to my mind is building attachment if `file` is not valid, not exists. This will probably cause an error. I would suggest @radiokills create another ticket for that?